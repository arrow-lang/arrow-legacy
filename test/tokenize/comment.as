// This is also a comment
