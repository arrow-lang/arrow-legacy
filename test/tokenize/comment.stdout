2,1-2: end
